src/lib.rs

```diff
@@ -1,19 +1,39 @@
 //! # Oxc Resolver
 //!
-//! Node.js Module Resolution.
+//! Node.js [CommonJS][cjs] and [ECMAScript][esm] Module Resolution.
 //!
-//! All configuration options are aligned with [enhanced-resolve].
+//! A module resolution is the process of finding the file referenced by a module specifier in
+//! `import "specifier"` or `require("specifier")`.
```

Comment:
Awesome! It's good to frame the name "specifier" immediately, since it's a very specific term.